src/app/cli/src/client.ml

```diff
@@ -82,6 +81,41 @@ let get_balance =
          | Error e ->
              printf "Failed to get balance %s\n" (Error.to_string_hum e) ))
 
+let get_trust_status =
+  let open Command.Param in
+  let open Deferred.Let_syntax in
+  let address_flag =
+    flag "ip-address"
+      ~doc:
+        "An IPv4 or IPv6 address for which you want to query the trust status"
```

Comment:
the first word in this string should be 'IP' - the formatting of the help strings uses the first word as a standin value.

run with -help to see what I mean.

Reply:
Fixed.